utils/openwrt-exporter/files/openwrt-exporter.init

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh /etc/rc.common
+# Copyright (C) 2013-2017 OpenWrt.org
+
+START=60
+USE_PROCD=1
+
+start_service() {
```

Comment:
Why don't we need a stop_service() and restart function? Have you tested stopping and restarting the service at runtime successfully?